code/unit_tests/map_tests.dm

```diff
@@ -264,18 +264,20 @@
 /datum/unit_test/storage_map_test/start_test()
 	var/bad_tests = 0
 
-	for(var/datum/storage/storage in world)
+// We have to ifdef this because _test_storage_items doesn't exist when UNIT_TEST isn't defined.
+#ifdef UNIT_TEST
```

Comment:
i think maybe if UNIT_TEST isn't defined this should always fail, since you can run map tests manually. that or make it fall back to `in world`